libs/components/pages/src/lib/modules/page/page.component.ts

```diff
@@ -52,23 +58,33 @@ export class SkyPageComponent implements OnInit, OnDestroy {
 
   #themeAdapter = inject(SkyPageThemeAdapterService);
   #layoutHostSvc = inject(SkyLayoutHostService);
+  #mediaQueryService = inject(SkyResizeObserverMediaQueryService);
+  #elementRef = inject(ElementRef);
 
   public ngOnInit(): void {
     this.#themeAdapter.addTheme();
 
     this.#layoutHostSvc.hostLayoutForChild
       .pipe(takeUntil(this.#ngUnsubscribe))
-      .subscribe((args) => {
+      .subscribe((args: any) => {
         this.#layoutForChild = args.layout as SkyPageLayoutType;
         this.#updateCssClass();
       });
+
+    if (this.#mediaQueryService) {
```

Comment:
Do you need an `if` here? There's no `optional` flag on the `inject()` call, so it should always be set.